style.css

```diff
@@ -87,6 +91,111 @@ p {
   width: 80%;
 }
 
+.modal {
+  width: 90%;
+  display: none;
+  flex-direction: column;
+  margin: 2.6rem auto;
+  position: fixed;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #fff;
+  padding: 0.7rem;
+  border-radius: 1rem;
+  border: 1px solid #dfe1e6;
+}
+
+.modal.active {
+  display: flex;
+}
```

Comment:
- [ ] The modal window looks incomplete in the mobile version, I recommend using `overflow-y: scroll` for a better UI 👇🏽 

![image](https://github.com/MohamedRoshdy2021/mobile-first-project/assets/58642949/032d0999-8854-408b-a88f-4dbbdb337c54)